src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +73,4 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
-}
+}
```

Comment:
Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes.